### PR TITLE
Fixes change password save button hidden

### PR DESCRIPTION
**Pull request by vinnyjth**
### What does this PR do, or why is it needed?

Simulator Screen Shot - iPhone 11 - 2019-10-08 at 09 39 32

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apolloschurchapp/src/user-settings/ChangePassword.js [hunk 1]
```diff
@@ -23,6 +28,8 @@ const Footer = styled({
   justifyContent: 'flex-end',
 })(SafeAreaView);
 
+console.warn(`${StatusBar.currentHeight}`);
```
[the remark this answers is not in the document]
Comment (vinnyjth) [reply packages/apolloschurchapp/src/user-settings/ChangePassword.js]: 🙅‍♂ My bad.

**Comment by redreceipt** [comment]
android isn't quite right

Screen Shot 2019-10-10 at 9 08 09 AM

**Comment by redreceipt** [comment]
looks like it's not taking into account the google keyboard helper bar

**Comment by vinnyjth** [comment]
@redreceipt That's not the same screen as the one in this issue.

**Comment by redreceipt** [comment]
oh haha

**Comment by redreceipt** [comment]
I'm sure there's an issue for the other screen somewhere too
